test/tools/go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/golang/protobuf v1.5.2
 	golang.org/x/exp/typeparams v0.0.0-20221114191408-850992195362 // indirect
 	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616
+	golang.org/x/sys v0.3.0 // indirect
```

Comment:
same question here. Are we adding this import because its required for the upgrade?